httpcore5/src/test/java/org/apache/hc/core5/ssl/TestSSLContextBuilder.java

```diff
@@ -27,6 +27,19 @@
 
 package org.apache.hc.core5.ssl;
 
+import org.apache.hc.core5.util.Timeout;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLException;
```

Comment:
@veselov Please avoid moving imports around. It just causes unnecessary noice in the chanage-set. Looks good otherwise.

Reply:
Done.